test/mock-api/header_test.js

```diff
@@ -2,10 +2,11 @@ var sinon = require("sinon");
 var should = require("should");
 var assert = require("assert");
 var helpers = require("./helpers");
+var axios = require("axios");
```

Comment:
Looks like a mistaken import?

Reply:
Ah yes mistaken import. Removed in latest commit.